### PR TITLE
ADEN-11546 Add tracking to the replay button in Porvata player

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

File: src/ad-products/templates/interface/video/replay-overlay.ts [hunk 1]
```diff
@@ -73,6 +74,11 @@ export class ReplayOverlay {
 		) {
 			const replayIcon = addReplayIcon(overlay);
 
+			communicationService.emit(eventsRepository.AD_ENGINE_VIDEO_REPLAY_DISPLAYED, {
```
Comment (nandy-andy) [inline src/ad-products/templates/interface/video/replay-overlay.ts:77]: Another way of doing it but maybe a bit challenging is to send the tracking data within the listener we setup in: https://github.com/Wikia/ad-engine/blob/f3466c06901613a7eec2d57cb6c8bd1556246171/src/ad-products/templates/interface/video/replay-overlay.ts#L59 🤔

File: src/ad-products/templates/interface/video/replay-overlay.ts [hunk 2]
```diff
@@ -92,6 +93,11 @@ export class ReplayOverlay {
 			container.parentElement.insertBefore(overlay, container);
 		}
 
+		communicationService.emit(eventsRepository.AD_ENGINE_VIDEO_REPLAY_OVERLAY_DISPLAYED, {
```
Comment (nandy-andy) [inline src/ad-products/templates/interface/video/replay-overlay.ts:96]: Another way of doing it but maybe a bit challenging is to send the tracking data within the listener we setup in: https://github.com/Wikia/ad-engine/blob/f3466c06901613a7eec2d57cb6c8bd1556246171/src/ad-products/templates/interface/video/replay-overlay.ts#L59 🤔

**Review by szwajkowski — APPROVED** [review]
Well, it looks like a copy of Learn more tracking, so it is gonna work :)

**Review by jakubbanas — APPROVED** [review]

File: platforms/shared/templates/handlers/video/video-bootstrap-handler.ts [hunk 3]
```diff
@@ -59,6 +59,7 @@ export class VideoBootstrapHandler implements TemplateStateHandler {
 
 			fromEvent(player, 'wikiaAdStarted').pipe(
 				mergeMap(() => fromEvent(player, 'wikiaAdCompleted')),
+				debounceTime(10),
```
Comment (szwajkowski) [inline platforms/shared/templates/handlers/video/video-bootstrap-handler.ts:62]: nit pick - 10 is a magic number - would be nice to make it a constant with indication whether its seconds or microseconds

**Review by szwajkowski — APPROVED** [review]
nice hack :D

**Pull request merged**